tests/conftest.py

```diff
@@ -42,3 +43,15 @@ def test_db(django_db_setup, django_db_blocker):
 @pytest.fixture
 def db_keys():
     return _keys
+
+
+@pytest.fixture(scope="module")
+def vcr_config():
+    return {
+        "ignore_localhost": True,
+    }
+
+
+@pytest.fixture(scope="module")
+def vcr_cassette_dir(request):
+    return os.path.join("tests", "data", "cassettes", request.module.__name__)
```

Comment:
Switched it over to ``pathlib``. I kept the scope to module for this one though, since the tests were failing when we tried to access request.module from a session scope.

Reply:
yep, that makes sense. Nice job specifying the location, I like the organization.